test/noncommercial.js

```diff
@@ -24,6 +24,7 @@ const util = include('test/utility.js');
 const factory = require('unionized');
 
 const noncommercialInput = include('test/data/testInputNoncommercial.json');
+const testURL = '/permits/applications/special-uses/noncommercial/';
```

Comment:
Glad we got this moved into a variable since all of the tests are using the same base route in this file.